frontend/packages/employee-frontend/src/components/common/Filters.tsx

```diff
@@ -276,6 +276,38 @@ export const UnitFilter = React.memo(function UnitFilter({
   )
 })
 
+interface FinanceDecisionManagerFilterProps {
+  employees: { id: string; label: string }[]
+  selected?: { id: string; label: string }
+  select: (employee: string) => void
+}
+
+export const FinanceDecisionManagerFilter = React.memo(
+  function FinanceDecisionManagerFilter({
+    employees,
```

Comment:
I think a more descriptive name like `decisionHandlers` would be better than a very generic `employees`.